tests/serial/mapping-greedy-kernel/GreedyDirection1.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<precice-configuration>
+  <data:scalar name="DataOne" />
+
+  <mesh name="MeshOne" dimensions="3">
+    <use-data name="DataOne" />
+  </mesh>
+
+  <mesh name="MeshTwo" dimensions="3">
+    <use-data name="DataOne" />
+  </mesh>
+
+  <participant name="SolverOne">
+    <provide-mesh name="MeshOne" />
+    <write-data name="DataOne" mesh="MeshOne" />
+  </participant>
+
+  <participant name="SolverTwo">
+    <receive-mesh name="MeshOne" from="SolverOne" />
+    <provide-mesh name="MeshTwo" />
+    <mapping:rbf-greedy
```

Comment:
Maybe an additional attribute for the tolerance (fMax) ? Can we give a default for that?

Reply:
Added parameters for a maximum number of iterations and the tolerance.